### ISSUE TITLE
Add support for TextField extensions 

### ISSUE THREAD

**Issue by mgarciavaadin** [vaadin/form-filler-addon#40]
IntegerField
EmailField
....

**Comment by czp13** [comment]
Not strictly related to this but:

ComponentUtils could support Date as well here:

```
  try {

                    Field field = itemClass.getDeclaredField(propName);
                    field.setAccessible(true);
                    if (field.getType().equals(LocalDate.class))
                        field.set(item, LocalDate.parse(propValue.toString(), DateTimeFormatter.ofPattern("yyyy-MM-dd")));
                    else if (field.getType().equals(LocalTime.class))
                        field.set(item, LocalTime.parse(propValue.toString(), DateTimeFormatter.ofPattern("HH:mm:ss")));
                    else if (field.getType().equals(LocalDateTime.class))
                        field.set(item, LocalDateTime.parse(propValue.toString(), DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss")));
                    else if (field.getType().equals(Double.class))
                        field.set(item, Double.valueOf(propValue.toString()));
                    else if (field.getType().equals(Integer.class))
                        field.set(item, Integer.valueOf(propValue.toString()));
                    else if (field.getType().equals(Long.class))
                        field.set(item, Long.valueOf(propValue.toString()));
                    else if (field.getType().equals(Float.class))
                        field.set(item, Float.valueOf(propValue.toString()));
                    else if (field.getType().equals(Boolean.class))
                        field.set(item, Boolean.valueOf(propValue.toString()));
                    else
                    field.set(item, propValue);

                } catch (Exception e) {
                    logger.error("Failed to set field value for '{}': {}", propName, e.getMessage());
                }
            }
```

**Comment by mgarciavaadin** [comment]
Closed as fixed in PR #50
